docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+---
+title: Host Certificates with Machine ID
+description: Issuing Host Certificates with Machine ID
+---
+
+In this guide, you will learn how to create host certificates through Machine ID for OpenSSH servers. This allows for the benefits of
```

Comment:
The primary use case here is to allow Teleport (human or machine) users to connect to OpenSSH servers, right?